compute/sendgrid/src/main/java/com/example/compute/sendgrid/SendEmailServlet.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.compute.sendgrid;
+
+import com.sendgrid.SendGrid;
+import com.sendgrid.SendGridException;
+
+// [START example]
+public class SendEmailServlet {
+  final static String SENDGRID_API_KEY = "YOUR-SENDGRID-API-KEY";
+  final static String SENDGRID_SENDER = "YOUR-SENDGRID-SENDER";
```

Comment:
Is the value for sendgrid-sender an email address? Maybe something like "SENDER-EMAIL" to be consistent with "DESTINATION-EMAIL"?